paasta_tools/kubernetes/bin/paasta_secrets_sync.py

```diff
@@ -215,5 +232,84 @@ def sync_secrets(
     return True
 
 
+def sync_boto_secrets(
```

Comment:
this should probably take a namespace param (just like `sync_secrets`) - otherwise we'll need to come back and add that in once we've added support for mounting secrets as files in task_proc/tron :)